.github/workflows/ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Configure CMake
         run: cmake --preset windows-default
       - name: Build
-        run: cmake --build _build
+        run: cmake --build _build -j4
```

Comment:
Better yet `-j"$(nproc)"` like other build scripts do.

Reply:
Oh, wait, this is Windows. Is this a cmd shell then? I wonder if `%NUMBER_OF_PROCESSORS%` would work in here?

Reply:
Sounds like it's powershell, so `-j $env:NUMBER_OF_PROCESSORS` then.

Reply:
I applied a quick heuristic here of corecount+2, similar to what ninja does by default.

Reply:
Sure, that's fine. The issue I have is with hard-coding the number of cores the CI system has, so it might change in the future without us noticing. You can add 2 to it, or multiple the number cores by some percentage, etc.

`-j [int]$env:NUMBER_OF_PROCESSORS+2`, with a space between, should do the trick then.

(I see [Bitcoin also uses `$env:NUMBER_OF_PROCESSORS` in their CI](https://github.com/bitcoin/bitcoin/blob/v26.0/.github/workflows/ci.yml#L337), so I expect this to work)

Reply:
`'-j' invalid number '[int]4+2' given.`

Reply:
Looks like `$([int]$env:NUMBER_OF_PROCESSORS+2)` works. Sorry for the incomplete tip, not very familiar with a powershell.